Tests/SonarScanner.MSBuild.Tasks.IntegrationTests/TargetsTests/RazorTargetTests.cs

```diff
@@ -107,6 +127,45 @@ public void Razor_ExcludedProject_NoErrorLog()
             AssertExpectedErrorLog(result, string.Empty);
         }
 
+        [TestMethod]
+        public void Razor_RazorSpecificOutputAndProjectInfo_AreCopiedToCorrectFolders()
+        {
+            // Arrange
+            var root = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
+            var projectSpecificOutDir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext, "0");
+            var temporaryProjectSpecificOutDir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext, @"0.tmp");
+            var razorSpecificOutDir = Path.Combine(root, "0.Razor");
+            var protoBufFile = Path.Combine(temporaryProjectSpecificOutDir, "ProtoBuf.txt");
+            var issues = Path.Combine(projectSpecificOutDir, "Issues.json");
+            File.WriteAllText(protoBufFile, string.Empty);
+            File.WriteAllText(issues, string.Empty);
```

Comment:
This test is hard to read without debugging, because lifecycle of `Issues.json` and `ProtoBuf.txt` isn't clear. Both normal build and Razor build will have issues and protobufs after all.

Let's be explicit and also add assertion for both files at the end.
```suggestion
            TestUtils.CreateEmptyFile(temporaryProjectSpecificOutDir, "Issues.FromMainBuild.json");
            var razorIssues = TestUtils.CreateEmptyFile(projectSpecificOutDir, "Issues.FromRazorBuild.json");
```